lib/sphero.js

```diff
@@ -24,6 +24,16 @@ function classCallCheck(instance, Constructor) {
   }
 }
 
+/**
+ * Creates a new sphero instance
+ * @constructor
+ * @param {String} address of the connected sphero
+ * @param {Object} opts for sphero setup
+ * @example
+ * var sphero = require("sphero");
+ * var spheroRPB = sphero("/dev/rfcomm0", { timeout: 300 });
```

Comment:
@stewart can you help with that? for windows it would be just `COM1` `COM2` etc...